[2025-03-11T06:55:18.593+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.fetch_purchases manual__2025-03-11T06:55:15+00:00 [queued]>
[2025-03-11T06:55:18.604+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.fetch_purchases manual__2025-03-11T06:55:15+00:00 [queued]>
[2025-03-11T06:55:18.604+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-11T06:55:18.640+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): fetch_purchases> on 2025-03-11 06:55:15+00:00
[2025-03-11T06:55:18.651+0000] {standard_task_runner.py:57} INFO - Started process 2193 to run task
[2025-03-11T06:55:18.656+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ingest_bronze_layer_dag', 'fetch_purchases', 'manual__2025-03-11T06:55:15+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/ingesting_pipeline.py', '--cfg-path', '/tmp/tmpxixda9bn']
[2025-03-11T06:55:18.661+0000] {standard_task_runner.py:85} INFO - Job 97: Subtask fetch_purchases
[2025-03-11T06:55:18.723+0000] {task_command.py:416} INFO - Running <TaskInstance: ingest_bronze_layer_dag.fetch_purchases manual__2025-03-11T06:55:15+00:00 [running]> on host 41ccd9d1769c
[2025-03-11T06:55:18.906+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andrew Tran' AIRFLOW_CTX_DAG_ID='ingest_bronze_layer_dag' AIRFLOW_CTX_TASK_ID='fetch_purchases' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T06:55:15+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T06:55:15+00:00'
[2025-03-11T06:55:18.910+0000] {base.py:73} INFO - Using connection ID 'postgres_source_db' for task execution.
[2025-03-11T06:55:18.915+0000] {logging_mixin.py:154} INFO - Incremental time: 1900-01-01 00:00:00
[2025-03-11T06:55:18.931+0000] {warnings.py:109} WARNING - /opt/***/dags/ingesting_pipeline.py:62: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)

[2025-03-11T06:55:18.949+0000] {logging_mixin.py:154} INFO - DataFrame:             timestamp  ...                     extracted_at
0 2022-04-01 22:53:05  ... 2025-03-11 06:55:18.931792+00:00
1 2022-04-01 22:53:05  ... 2025-03-11 06:55:18.931792+00:00
2 2022-04-01 19:03:22  ... 2025-03-11 06:55:18.931792+00:00
3 2022-04-03 17:17:38  ... 2025-03-11 06:55:18.931792+00:00
4 2022-04-02 11:53:15  ... 2025-03-11 06:55:18.931792+00:00

[5 rows x 7 columns]
[2025-03-11T06:55:18.952+0000] {base.py:73} INFO - Using connection ID 'postgres_warehouse_db' for task execution.
[2025-03-11T06:55:18.952+0000] {logging_mixin.py:154} INFO - URI: postgresql://***:***@postgres:5432/warehouse_db?__extra__=%7B%7D
[2025-03-11T06:55:18.956+0000] {base.py:73} INFO - Using connection ID 'postgres_warehouse_db' for task execution.
[2025-03-11T06:55:18.966+0000] {python.py:194} INFO - Done. Returned value was: Inserted 76 rows into bronze_purchases
[2025-03-11T06:55:18.979+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ingest_bronze_layer_dag, task_id=fetch_purchases, execution_date=20250311T065515, start_date=20250311T065518, end_date=20250311T065518
[2025-03-11T06:55:19.011+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-11T06:55:19.026+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
